### PR TITLE
Fix htsjdk deprecation warnings.

### PR DESCRIPTION
**Pull request by tomwhite**

### CODE REVIEW

File: src/main/java/org/seqdoop/hadoop_bam/BCFRecordWriter.java [hunk 1]
```diff
@@ -106,8 +104,8 @@ private void init(
 		final BCFStoppableOutputStream stopOut =
 			new BCFStoppableOutputStream(!writeHeader, output);
 
-		writer = VariantContextWriterFactory.create(
-			stopOut, null, EnumSet.of(Options.FORCE_BCF));
+		writer = new VariantContextWriterBuilder().clearOptions()
+				.setOutputBCFStream(stopOut).build();
```
Comment (AndreSchumacher) [inline src/main/java/org/seqdoop/hadoop_bam/BCFRecordWriter.java]: It seems this does not actually set the FORCE_BCF option? As far as I see it probably will not matter much but in order to not introduce any side effects should we maybe set it via setOption?

**Comment by AndreSchumacher** [comment]
@tomwhite thanks for cleaning up those warnings. I added one comment above. Otherwise looks good to me, feel free to merge.

File: src/main/java/org/seqdoop/hadoop_bam/BCFRecordWriter.java (hunk 1 above)
Comment (tomwhite) [inline src/main/java/org/seqdoop/hadoop_bam/BCFRecordWriter.java]: Fixed.

**Comment by tomwhite** [comment]
Thanks for taking a look @AndreSchumacher!

**Pull request merged**